settings.py

```diff
@@ -57,6 +57,7 @@
     ('fr', 'French'),
     ('fra', 'French'),  # we need this alias
     ('hin', 'Hindi'),
+    ('sw', 'Swahili'),
```

Comment:
I think HQ uses the 3-letter codes mostly - are you sure this is the right one? the other one is swa http://en.wikipedia.org/wiki/Swahili_language